makeflow/src/makeflow.c

```diff
@@ -539,6 +631,12 @@ static int makeflow_node_ready(struct dag *d, struct dag_node *n)
 	if(n->state != DAG_NODE_STATE_WAITING)
 		return 0;
 
+		 if(makeflow_is_local_node(n)){
```

Comment:
Correct the indentation here. It confuses that it is a separate check from checking if the node is waiting.